.github/scripts/template-zip-autogen.sh

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/env bash
 set -x
 
+if git diff HEAD^ ./templates/package.json | grep version
+    then 
```

Comment:
Please reformat, indent is incorrect.

BTW, to maintain this script's function focus on zip generation. Can we move this section for version check in another place, like cd.yml?

Reply:
sure, deleted.